test/unit/test_query_context_cache.py

```diff
@@ -51,6 +51,35 @@ def reset_data(self):
         self.priorities = [BASE_PRIORITY + i for i in range(self.capacity)]
 
 
+# This function is used for testing the deserialize_to_dict function is correct. Note that `QueryContextCache::serialize_to_dict` function is used in sending the query context back to GS.
+# It has slight difference with this function on `context` field.
+def serialize_to_json(qcc: QueryContextCache) -> str:
+    with qcc._lock:
+        qcc.log_cache_entries()
+
+        if len(qcc._tree_set) == 0:
+            return ""
+
+        try:
+            data = {
+                "entries": [
+                    {
+                        "id": qce.id,
+                        "timestamp": qce.read_timestamp,
+                        "priority": qce.priority,
+                        "context": qce.context,
+                    }
+                    for qce in qcc._tree_set
+                ]
+            }
+            # Serialize the data to JSON
+            serialized_data = json.dumps(data)
+
+            return serialized_data
+        except Exception as e:
+            return ""
```

Comment:
```suggestion
        except Exception:
            return ""
```
This is causing lint issues. check with this change

Reply:
Yeah. This should be removed since `e` is not used here.